dataeng/resources/load-warehouse-bigquery.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+if [ "x$NUM_REDUCE_TASKS" = "x" ]; then
+    NUM_REDUCE_TASKS=$(( $NUM_TASK_CAPACITY*2 ))
+fi
```

Comment:
I'd argue that *this* should be removed, because it's not passed to the task below at all, and therefore serves no function.